specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/examples/licenseProfile/LicenseProfile_CreateOrUpdate.json

```diff
@@ -0,0 +1,100 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "machineName": "myMachine",
+    "licenseProfileName": "default",
+    "api-version": "2023-06-20-preview",
+    "parameters": {
+      "location": "eastus2euap",
+      "properties": {
+        "esuProfile": {
+          "arcEsuEnrolled": "false",
+          "esuKeys": [
+            {
+              "esuSku": "esuSkuNumber1",
+              "licenseStatus": "licenseStatus1"
+            },
+            {
+              "esuSku": "esuSkuNumber2",
+              "licenseStatus": "licenseStatus2"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/Machines/myMachine/licenseProfiles/default",
+        "name": "default",
+        "type": "Microsoft.HybridCompute/machines/licenseProfiles",
+        "location": "eastus2euap",
+        "tags": null,
+        "properties": {
+          "esuProfile": {
+            "arcEsuEnrolled": "false",
+            "esuKeys": [
+              {
+                "esuSku": "esuSkuNumber1",
+                "licenseStatus": "licenseStatus1"
+              },
+              {
+                "esuSku": "esuSkuNumber2",
+                "licenseStatus": "licenseStatus2"
+              }
+            ],
+            "esuEnrollmentHistory": [
+              {
+                "oldState": "oldState",
+                "newState": "newState",
+                "time": "2020-08-08T20:42:10.999Z"
+              },
+              {
+                "oldState": "oldState",
+                "newState": "newState",
+                "time": "2020-08-08T20:42:10.999Z"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
```

Comment:
the async response wouldn't have the body info, just the callback details

Reply:
also, why 201 here and 202 for the update example?

Reply:
ARM restriction